controllers/rollingupgrade_controller.go

```diff
@@ -444,6 +444,7 @@ func (r *RollingUpgradeReconciler) getNodeName(i *autoscaling.Instance, nodeList
 	return node.Name
 }
 
+//TODO(sbadiger): Why are we listing all the nodes here and not just the nodes from ASG? Use asg.Instances instead of nodeList
```

Comment:
Looks like a leftover comment from design/dev phase. Let's nuke it.

Reply:
Yes and No. **nodeList** has all the nodes out of ASG which will always fail the if condition inside the function. Maybe we can track this through another issue.

Reply:
Removed the comment.